src/libs/blueprint/conduit_blueprint_mesh_examples.hpp

```diff
@@ -267,6 +267,25 @@ namespace examples
                                      conduit::index_t dz,
                                      conduit::Node &res);
 
+    /// Generates a multidomain grid of examples.  spec is a list or object
+    /// specifying one or more domains.  Each child of spec represents a domain
+    /// in either two or three dimensions.  Each child has
+    ///  - integral children npts_x, npts_y, optionally npts_z
+    ///  - integral child domain_id
+    ///  - numerical array children corner_xs, corner_ys, optionally corner_zs
+    ///
+    /// The children in each domain specification represent the locations of
+    /// the corners of that domain and contain two or three coordinates.
+    void CONDUIT_BLUEPRINT_API bentgrid(const conduit::Node& spec,
+                                        conduit::Node& res);
+
+    /// Generates a braid-like example mesh that covers elements defined in a
+    /// rectilinear grid.  Currently hexs are supported for 3D meshes and quads
```

Comment:
This actually creates an unstructured grid, for this example we want to target structured 
(still explicit coords, but with an implicit structured topology)